src/index.js

```diff
@@ -101,15 +105,6 @@ function fireChangeEvent(event) {
   event.target.removeEventListener("blur", fireChangeEvent);
 }
 
-function blurFocusedElement(element, focusedElement, wasAnotherElementFocused) {
```

Comment:
Now that we focus before click, we don't need to blur elements at the end of the actions.